qkrehdwns/likelion-assignment-crud/src/main/java/likelion/likelionassignmentcrud/controller/CityController.java

```diff
@@ -0,0 +1,38 @@
+package likelion.likelionassignmentcrud.controller;
+
+import likelion.likelionassignmentcrud.domain.City;
+import likelion.likelionassignmentcrud.dto.city.CityResponseDto;
+import likelion.likelionassignmentcrud.dto.city.CitySaveDto;
+import likelion.likelionassignmentcrud.service.CityService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.List;
+
+@Controller
```

Comment:
현재 컨트롤러에 @Controller가 사용되고 있는데 이 상황에서는 REST API를 반환하고 있기 때문에 @RestController가 더 적절해요!

```suggestion
@RestController
```